app/views/revisions/sidebar.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @issue %>
```

Comment:
I imagine we have the `@record` here. If so, we can probably simplify this file to:

```
<%= render "#{@record.class.name}/sidebar" %>
```

Reply:
That won't work because we need to render `nodes/sidebar` for notes and evidence